apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -65,8 +68,63 @@ DropletAutocompleteParameterTooltipManager.prototype.onCursorMovement_ = functio
     return;
   }
 
+  if (this.startingAutoComplete) {
+    // Guard against re-entrancy that occurs inside the showParamDropdownIfNeeded_() below
```

Comment:
Yes, nasty recursion occurs. Even if I block calling startAutocomplete again, the call to `this.updateParameterTooltip_` also causes us problems. So we just need to bail out if we're in the midst of starting autocomplete.